packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.base.tsx

```diff
@@ -1,27 +1,42 @@
 import * as React from 'react';
+
+import { FocusZone, FocusZoneDirection, IFocusZone } from '../../FocusZone';
+import { IKeytipProps } from '../../Keytip';
 import {
-  css,
   BaseComponent,
+  classNamesFunction,
   createRef,
-  getNativeProps,
+  css,
+  customizable,
   divProperties,
+  elementContains,
   focusFirstChild,
-  elementContains
+  getNativeProps,
+  IClassNames
 } from '../../Utilities';
-import { mergeStyles } from '../../Styling';
-import { IOverflowSet, IOverflowSetProps, IOverflowSetItemProps } from './OverflowSet.types';
-import { IFocusZone, FocusZone, FocusZoneDirection } from '../../FocusZone';
 import { KeytipManager } from '../../utilities/keytips/KeytipManager';
-import { IKeytipProps } from '../../Keytip';
-import * as stylesImport from './OverflowSet.scss';
-
-const styles: any = stylesImport;
+import {
+  IOverflowSet,
+  IOverflowSetItemProps,
+  IOverflowSetProps,
+  IOverflowSetStyles,
+  IOverflowSetStyleProps
+} from './OverflowSet.types';
+
+const getClassNames = classNamesFunction<IOverflowSetStyleProps, IOverflowSetStyles>();
+
+@customizable('OverflowSet', [])
```

Comment:
I'm not sure this is right.

Reply:
Theme doesn't seem to be used anywhere in styling for this component, but I don't know if customizable provides any other functionality that makes it required here...  `Customizations.getSettings()` seems to get some props too (within `customizable.render()`), but I'm not sure what they are.